packages/babel-plugin-transform-block-scoping/test/fixtures/regression/rename-with-other-plugins/input.ts

```diff
@@ -0,0 +1,15 @@
+// `index` will be registered into `scope.globals`
+type SomeLookup = {
+  [index: number]: any[];
+};
```

Comment:
I don't think this test is very good, but I don't know how to make it better.
Because after we change `traverse` it will not be registered in `globals`.

Reply:
@liuxingbaoyu Don't forget about this!